core/src/com/group66/game/shop/BuyScoreMultiplier.java

```diff
@@ -26,8 +26,8 @@ public BuyScoreMultiplier() {
          */
         public void buy(BuyStateInstance instance, DynamicSettings dynamicSettings) {
```

Comment:
Yes we need to. Because there could be multiple instances of dynamicSettings. Its not a singleton.